app/assets/stylesheets/maps.css.scss

```diff
@@ -33,10 +33,20 @@ html, body {
 .map-list .map img {
   max-height:300px;
 }
+
 .map-list .map h3 {
   overflow:hidden;
 }
 
+.map-list .map {
+  clear:none;
+  display: inline-block
+}
+
+.map-list div:nth-child(4n+1){
```

Comment:
😮😮😮😮😮 🎉 

This is super! 

Would it be possible to apply this to these styles as well? 

https://github.com/publiclab/plots2/blob/master/app/views/grids/_thumbnail.html.erb

Reply:
Yes, I will look into them!!!